src/onboarding/templates/multisig/index.js

```diff
@@ -1,11 +1,72 @@
+import { isAddress } from 'web3-utils'
+import ConfigureMultisigToken from './ConfigureMultisigToken'
+import ConfigureMultisigAddresses from './ConfigureMultisigAddresses'
 import icon from './assets/icon.svg'
 
 const template = {
   name: 'multisig',
   label: 'Token project with Multisig',
   icon,
-  screens: [],
-  prepareData: () => ({}),
+  screens: [
+    {
+      screen: 'multisig-addresses',
+      fields: {
+        signers: {
+          defaultValue: () => [''],
+          filter: value => {
+            if (!Array.isArray(value)) {
+              return ['']
+            }
+            if (value.length < 1) return ['']
+            return value
+          },
+        },
+        neededSignatures: {
+          defaultValue: () => 1,
+          filter: (value, { signers }) => {
+            const intValue = parseInt(value, 10)
+            return isNaN(intValue)
+              ? 1
+              : Math.min(signers.length, Math.max(1, value))
+          },
+        },
+      },
+      validateScreen: ({ signers, neededSignatures }) => {
+        if (!signers.every(signer => isAddress(signer.trim()))) {
```

Comment:
Just return the `every()` :).

Reply:
Ah yes thanks, I was testing `neededSignatures` here before! (now I onlyfilter it in `prepareData`)